addon/src/-private/array.ts

```diff
@@ -55,6 +55,7 @@ function convertToInt(prop: number | string | symbol): number | null {
   return num % 1 === 0 ? num : null;
 }
 
+// eslint-disable-next-line @typescript-eslint/no-unsafe-declaration-merging
```

Comment:
Hmm... what is this even merging with? Is this actually necessary?

Reply:
At the bottom of the file. It is needed ya. The way we make these things look like normal arrays is the declaration merge.

Object is doing the same thing